### ISSUE TITLE
Disable nominatim replication on offline server

### ISSUE THREAD

**Issue by hamid-moghadam** [mediagis/nominatim-docker#257]
Hi mediagis. Thanks for your repo.

Currently I'm working on a server that is not connected to the internet and it should be used as a standalone reverse geocoding server for servicing other applications.

**Problem**:
In [init.sh](https://github.com/mediagis/nominatim-docker/blob/4c8b2699f8c01b795bce448e81889ac59d08c14e/3.7/init.sh) There's a "nominatim utility script" that connect to "REPLICATION_URL" or "openstreetmap.org" in order to update database; So I get the error because there's no internet connection!

**Question**:
What if adding an environment variable for `docker run` in order to ignore this command in `init.sh`:
https://github.com/mediagis/nominatim-docker/blob/4c8b2699f8c01b795bce448e81889ac59d08c14e/3.7/init.sh#L56

Is it possible to ignore any updates and just pass .pbf file and go on? Is there anywhere that should be considered in order to achieve offline mode?

**Comment by fenrew** [comment]
I'm currently facing the exact same problem with our offline Nominatim server and would love to have the option to turn it off

**Comment by leonardehrenfried** [comment]
I would be ok with a PR which makes this an optional env param. So if `REPLICATION_URL` is not defined we could skip the line initialising the replication.

Would you like to contribute this?

**Comment by leonardehrenfried** [comment]
(I had no idea that this line connects to the URL. I thought that it just sets up the database for replication.)

**Comment by hamid-moghadam** [comment]
I'm interested in contributing to it.

Take a look at these outputs:
```
+ sudo -u nominatim nominatim admin --check-database
2021-09-13 09:30:32: Using project directory: /nominatim
2021-09-13 09:30:32: Checking database
Checking database connection ... OK
Checking for placex table ... OK
Checking for placex content ... OK
Checking that nominatim.so module is installed ... OK
Checking indexing status ... OK
Checking that database indexes are complete ... OK
Checking that all database indexes are valid ... OK
Checking TIGER external data table. ... not applicable
+ sudo -E -u nominatim nominatim replication --init
2021-09-13 09:30:32: Using project directory: /nominatim
2021-09-13 09:30:32: Initialising replication updates
2021-09-13 09:33:32: Failed to load URL: https://www.openstreetmap.org/api/0.6/n ode/8350237786/1
Traceback (most recent call last):
File "/usr/lib/python3.8/urllib/request.py", line 1354, in do_open
h.request(req.get_method(), req.selector, req.data, headers,
File "/usr/lib/python3.8/http/client.py", line 1252, in request
self._send_request(method, url, body, headers, encode_chunked)
File "/usr/lib/python3.8/http/client.py", line 1298, in _send_request
self.endheaders(body, encode_chunked=encode_chunked)
File "/usr/lib/python3.8/http/client.py", line 1247, in endheaders
self._send_output(message_body, encode_chunked=encode_chunked)
File "/usr/lib/python3.8/http/client.py", line 1007, in _send_output
self.send(msg)
File "/usr/lib/python3.8/http/client.py", line 947, in send
self.connect()
File "/usr/lib/python3.8/http/client.py", line 1414, in connect
super().connect()
File "/usr/lib/python3.8/http/client.py", line 918, in connect
self.sock = self._create_connection(
File "/usr/lib/python3.8/socket.py", line 808, in create_connection
raise err
File "/usr/lib/python3.8/socket.py", line 796, in create_connection
sock.connect(sa)
OSError: [Errno 99] Cannot assign requested address

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "/usr/local/bin/nominatim", line 11, in <module>
exit(cli.nominatim(module_dir='/usr/local/lib/nominatim/module',
File "/usr/local/lib/nominatim/lib-python/nominatim/cli.py", line 288, in nomi natim
return parser.run(**kwargs)
File "/usr/local/lib/nominatim/lib-python/nominatim/cli.py", line 92, in run
return args.command.run(args)
File "/usr/local/lib/nominatim/lib-python/nominatim/clicmd/replication.py", li ne 149, in run
return UpdateReplication._init_replication(args)
File "/usr/local/lib/nominatim/lib-python/nominatim/clicmd/replication.py", li ne 55, in _init_replication
replication.init_replication(conn, base_url=args.config.REPLICATION_URL)
File "/usr/local/lib/nominatim/lib-python/nominatim/tools/replication.py", lin e 27, in init_replication
date = status.compute_database_date(conn)
File "/usr/local/lib/nominatim/lib-python/nominatim/db/status.py", line 29, in compute_database_date
data = get_url(node_url)
File "/usr/local/lib/nominatim/lib-python/nominatim/tools/exec_utils.py", line 140, in get_url
with urlrequest.urlopen(urlrequest.Request(url, headers=headers)) as respons e:
File "/usr/lib/python3.8/urllib/request.py", line 222, in urlopen
return opener.open(url, data, timeout)
File "/usr/lib/python3.8/urllib/request.py", line 525, in open
response = self._open(req, data)
File "/usr/lib/python3.8/urllib/request.py", line 542, in _open
result = self._call_chain(self.handle_open, protocol, protocol +
File "/usr/lib/python3.8/urllib/request.py", line 502, in _call_chain
result = func(*args)
File "/usr/lib/python3.8/urllib/request.py", line 1397, in https_open
return self.do_open(http.client.HTTPSConnection, req,
File "/usr/lib/python3.8/urllib/request.py", line 1357, in do_open
raise URLError(err)
urllib.error.URLError: <urlopen error [Errno 99] Cannot assign requested address >
```

It's trying to get some data in openstreetmap and the last executing line is `sudo -E -u nominatim nominatim replication --init`.
I have no idea what's going on there and why it's trying to get a specific node from openstreetmap meanwhile I've attached a .osm.pbf file to it.

What do you think?

**Comment by leonardehrenfried** [comment]
Well, if the env `REPLICATION_URL` is not defined, simply skip this command in init.sh.

**Comment by leonardehrenfried** [comment]
But right now this param is required so you also have to remove the check.

**Comment by hamid-moghadam** [comment]
I've made some changes and tested them in online and offline mode.
Please check it out: #258 
Thanks

**Comment by leonardehrenfried** [comment]
Fixed by #270
